resources/css/login.css

```diff
@@ -0,0 +1,41 @@
+input[type=email], input[type=password] {
```

Comment:
please format the files

Reply:
formatted it... but this was the file created by the gulp compiler.

Reply:
then these should be in the .gitignore files